packages/orchestrator-ui-components/src/components/WfoSubscription/WfoInSyncField.tsx

```diff
@@ -30,20 +33,21 @@ export const WfoInSyncField = ({ subscriptionDetail }: WfoInSyncFieldProps) => {
         subscriptionDetail.processes.page,
     );
     const [setSubscriptionInSync] = useSetSubscriptionInSyncMutation();
+    const { isFetching } = useGetSubscriptionDetailQuery({
+        subscriptionId: subscriptionDetail.subscriptionId,
+    });
     const { showToastMessage } = useShowToastMessage();
     const { showConfirmDialog } = useContext(ConfirmationDialogContext);
 
     const setInSyncAction = () => {
         setSubscriptionInSync(subscriptionDetail.subscriptionId)
             .unwrap()
             .then(() => {
-                // Optimistic update for now
                 showToastMessage(
                     ToastTypes.SUCCESS,
                     t('setInSyncSuccess.text'),
                     t('setInSyncSuccess.title'),
                 );
-                subscriptionDetail.insync = true;
```

Comment:
Assigning value to read-only property triggered the catch and showing the error toast after the success toast.